app/src/main/java/org/projectbuendia/client/models/UnsyncForm.java

```diff
@@ -0,0 +1,86 @@
+// Copyright 2015 The Project Buendia Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License.  You may obtain a copy
+// of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distrib-
+// uted under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
+// specific language governing permissions and limitations under the License.
+
+package org.projectbuendia.client.models;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+
+import org.projectbuendia.client.providers.Contracts;
+import org.projectbuendia.client.utils.Utils;
+
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public final class UnsyncForm  {
```

Comment:
Can we use a name like "UnsentForm" or "PendingForm" instead? "UnsyncForm" isn't clear which direction the form is going in (client to server or server to client).